src/content/configuration/externals.md

```diff
@@ -144,26 +145,89 @@ This syntax is used to describe all the possible ways that an external library c
 
 `function (context, request, callback)`
 
-It might be useful to define your own function to control the behavior of what you want to externalize from webpack. [webpack-node-externals](https://www.npmjs.com/package/webpack-node-externals), for example, excludes all modules from the `node_modules` directory and provides some options too, for example, whitelist packages.
+It might be useful to define your own function to control the behavior of what you want to externalize from webpack. [webpack-node-externals](https://www.npmjs.com/package/webpack-node-externals), for example, excludes all modules from the `node_modules` directory and provides some options to, for example, whitelist packages.
```

Comment:
I know its been there but double `, for example,` isnt great, lets maybe to this?
```suggestion
It might be useful to define your own function to control the behavior of what you want to externalize from webpack. [webpack-node-externals](https://www.npmjs.com/package/webpack-node-externals), for example, excludes all modules from the `node_modules` directory and provides options to whitelist packages.
```

Reply:
That stood out to me, too, but was hesitant to change something that was there. I think your suggestion works well.